_posts/2019-10-25-Kickstarter-Trajectory-2019-Summer-Edition.md

```diff
@@ -0,0 +1,220 @@
+
+# Final day
+
+On the day after the final course day, all the Kickstarters gave a short presentation about themselves in front of the others and the management. 
+This way, everyone present got to know the others, with both their professional interests and achievements, as well as a little on the personal side. 
+Afterwards, there was a moment for networking and socializing with everyone, and an official graduation. 
+The next day, the DevCases started, which we’ll explain below!
+
+<div style="text-align: center;">
+  <img src="/img/2019-10-25-Kickstarter-Trajectory-2019-Summer-Edition/FinalDay.png" width="80%" height="80%" target="_blank">
+</div>
+
+# DevCases 
+
+The team of 8 JWorks Kickstarters was divided into 2 teams, and thus 2 DevCases:
+* Brecht, Imad, Lennert, Lore and Mohammed worked together on the Zero Plastic Rivers case for the University of Antwerp
+* Duncan, Jasper and Nicholas were set on the task of designing Chatbot Dina for internal use
+
+##  Zero Plastic Rivers
+
+<img class="image right" alt="ZeroPlasticRivers" src="/img/2019-10-25-Kickstarter-Trajectory-2019-Summer-Edition/ZeroPlasticRivers.png" width="40%" height="40%" target="_blank">
+
+At the end of the Kickstarter trajectory, we were asked to develop a web application to monitor the plastic as it travels through the Schelde. 
+For this purpose, we'd be using GPS trackers alongside QR-scanners.
+This application is aimed at a PhD carried out at the University of Antwerp that consists of visualizing the plastic flow through the entire river, from the basin to the mouth. 
+After visualizing it, an efficient remediation strategy could be made.
+
+The main objective of the application is to create a monitoring network to collect plastic waste, for example, in dams, locks or water treatment plants. 
+This way, plastic flows can be calculated for example in sub-basins or piers to estimate the total flow to the estuary.
+To activate this system, plastic bottles in the Schelde river will be released at different strategic points with GPS trackers and personalized labels. 
+These contain relevant information such as the identifier or the url to the application.
+
+The application consists of two parts.
+The first part is aimed at citizens who wish to help the cause, who can notify this surveillance network when they find a bottle as shown in the image on the right.
+The second part is aimed at the researchers, and could be seen as the "backend" of the project, where the data given by the GPS trackers and the citizens is visualized in a clear and orderly way.
+
+## Chatbot Dina
+
+In the second DevCase, we built chatbot Dina for the Fleet department of Ordina. 
+The Chatbot team set off using the Chatlayer bot framework, later to be joined by an implementation in Dialogflow. 
+Since chatbots are a relatively new technology, we wanted to keep options open and look for the best possible implementation. 
```

Comment:
```suggestion
Since chatbots are a relatively new technology, we wanted to keep our options open and look for the best possible implementation. 
```